filament/src/PostProcessManager.cpp

```diff
@@ -321,6 +321,8 @@ FrameGraphResource PostProcessManager::ssao(FrameGraph& fg, FrameGraphResource d
                 FMaterialInstance* const pInstance = mSSAOMaterialInstance;
                 pInstance->setParameter("depth", depth, params);
                 pInstance->setParameter("radius", data.options.radius);
+                pInstance->setParameter("invRadiusSquared", 1.0f / (data.options.radius * data.options.radius));
+                pInstance->setParameter("projectionScaleRadius", 500.0f * data.options.radius);
```

Comment:
Is there a specific reason for the 500f here? Maybe add a comment

Reply:
oh right -- this is temporary, this will be calculated enventually. it's just that 500 usually works.